packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -435,7 +469,7 @@ export class SafeDsPythonGenerator {
 
     private generateBlockLambda(blockLambda: SdsBlockLambda, frame: GenerationInfoFrame): CompositeGeneratorNode {
         const results = streamBlockLambdaResults(blockLambda).toArray();
-        let lambdaBlock = this.generateBlock(blockLambda.body, frame);
+        let lambdaBlock = this.generateBlock(blockLambda.body, frame, true);
```

Comment:
Why are we "insidePipeline" if we are inside a block lambda?

Reply:
We should only be "insidePipeline" if the block lambda is located inside a pipeline.
That is why the `generateLambda` parameter is set to `true` in this line, so the placeholder saving call is not generated.
To my understanding, the placeholders should only be saved when being assigned directly in the pipeline code.